Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -216,21 +216,14 @@ class PlaceDetailsCardViewController: UIViewController {
     }
 
     @objc private func autoMoveToNextCarouselImage() {
-        let pageSize = imageCarouselCollectionView.bounds.size
-        let contentWidth = imageCarouselCollectionView.contentSize.width
-        let oldOffset = imageCarouselCollectionView.contentOffset
-        let newX = (oldOffset.x + pageSize.width).truncatingRemainder(dividingBy: contentWidth)
-        let newOffset = CGPoint(x: newX, y: oldOffset.y)
-        self.imageCarouselCollectionView.setContentOffset(newOffset, animated: true)
+        let currentIndex = pageControl.currentPage
+        self.imageCarouselCollectionView.scrollToItem(at: IndexPath(item: currentIndex+1, section: 0), at: UICollectionViewScrollPosition.centeredHorizontally, animated: true)
```

Comment:
Since we're always incrementing the currentIndex this will go out of bounds when it reaches the end. If we want to keep with the looping back to the first image we should modulo this calculation (for some reason Swift got rid of % and uses truncatingRemainder)